tdrs-backend/reports/zap.conf

```diff
@@ -120,3 +120,5 @@
 90034	FAIL	(Cloud Metadata Potentially Exposed - Active/beta)
 100000	WARN	(A Client Error response code was returned by the server)
 100001	WARN	(Unexpected Content-Type was returned)
+100001	OUTOFSCOPE	http://web:8080/v1/login
```

Comment:
This prevents the `Unexpected Content Type` alert from triggering on `/v1/login` which always does a redirect to Login.gov and thus is expected to be an HTTP response.